openapi/anchor-platform/main.yaml

```diff
@@ -1,26 +1,241 @@
-openapi: 3.0.0
+openapi: 3.1.0
```

Comment:
Do we need to store all openapi stuff in `main.yaml` from now on?

Reply:
At the moment, yes. That's the "heart" of the specfile, kindof like the "index.html" page. In the future, we could certainly break it into a more manageable series of smaller files (similar to how Horizon is done (but also done better, that ones a bit of a rat's nest lol)) to get bundled together (like how `yarn api` bundles them already).

Reply:
Even if we did break things into sub-files, they will still need to get `$ref`ed in the `main.yaml` file in order to be bundled up.

Reply:
Yep I think that was how it was done before, right?